tests/conftest.py

```diff
@@ -0,0 +1,25 @@
+import json
+from pathlib import Path
+import pytest
+
+
+@pytest.fixture()
+def expected_files():
+    """Read fixture files from test/queries/expected/PLUGIN/QUERY and remove .txt suffix."""
+    from qenerate.core.plugin import GeneratedFile
```

Comment:
out of curiosity: why is this inlined instead of declared on top?

Reply:
I sometimes face import errors, but that's not the case here.